packages/client/tests/functional/0-legacy-ports/batch-find-unique/tests.ts

```diff
@@ -106,5 +107,12 @@ testMatrix.setupTestSuite(() => {
         },
       ]
     `)
-  })
-})
+    })
+  },
+  {
+    skipEngine: {
+      from: ['wasm'],
+      reason: 'Fails on init with `unwrap_throw` failed',
```

Comment:
Trying this locally, this yields `Expected two assertions to be called but received zero assertion calls` to me